tsdown.config.ts

```diff
@@ -1,10 +1,8 @@
-import { defineConfig } from 'tsup';
+import { defineConfig } from 'tsdown';
 
 export default defineConfig({
-  bundle: false,
-  clean: true,
+  unbundle: true,
   dts: true,
   entry: ['lib/**/*.ts'],
-  format: ['esm'],
```

Comment:
Are there any behavior changes from this tsup/tsdown conversion?

Reply:
No, it's meant to be a drop in replacement, just faster. If we were able to set `isolatedModules` to true in the tsconfig, it would be even faster. But the way the plugin is setup, we can't really do that, and still get rich type information. So, yeah, just swapping one for the other, won't change anything. I was able to remove some of those settings because tsdown will infer them from the package.json.